[I][2019-11-14T21:59:33+0000] Mode: LISTEN_TCP
[I][2019-11-14T21:59:33+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-14T21:59:33+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-14T21:59:33+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-14T21:59:33+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-14T21:59:33+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-14T21:59:33+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-14T21:59:33+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-14T21:59:33+0000] Listening on [::]:31337
[I][2019-11-14T21:59:45+0000] New connection from: [::ffff:172.18.0.1]:57118 on: [::ffff:172.18.0.2]:31337
[I][2019-11-14T21:59:45+0000] Executing '/var/task/main.py' for '[::ffff:172.18.0.1]:57118'
[E][2019-11-14T21:59:45+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-14T21:59:45+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-14T21:59:45+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-14T21:59:45+0000] pid=9 ([::ffff:172.18.0.1]:57118) exited with status: 255, (PIDs left: 0)
[I][2019-11-14T22:03:39+0000] Mode: LISTEN_TCP
[I][2019-11-14T22:03:39+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-14T22:03:39+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-14T22:03:39+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-14T22:03:39+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-14T22:03:39+0000][9] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-14T22:03:39+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-14T22:03:39+0000][9] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-14T22:03:39+0000] Listening on [::]:31337
[I][2019-11-14T22:04:15+0000] New connection from: [::ffff:172.19.0.1]:60204 on: [::ffff:172.19.0.2]:31337
[I][2019-11-14T22:04:15+0000] Executing '/var/task/main.py' for '[::ffff:172.19.0.1]:60204'
[E][2019-11-14T22:04:15+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-14T22:04:15+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-14T22:04:15+0000][9] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-14T22:04:15+0000] pid=28 ([::ffff:172.19.0.1]:60204) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:20:16+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:20:16+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:20:16+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:20:16+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:20:16+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:20:16+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:20:16+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:20:16+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:20:16+0000] Listening on [::]:31337
[I][2019-11-29T00:20:29+0000] New connection from: [::ffff:94.25.172.81]:27495 on: [::ffff:192.168.144.2]:31337
[I][2019-11-29T00:20:29+0000] Executing '/var/task/main.py' for '[::ffff:94.25.172.81]:27495'
[I][2019-11-29T00:20:29+0000] pid=9 ([::ffff:94.25.172.81]:27495) exited with status: 2, (PIDs left: 0)
[I][2019-11-29T00:26:07+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:26:07+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:26:07+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:26:07+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:26:07+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:26:07+0000][9] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:26:07+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:26:07+0000][9] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:26:07+0000] Listening on [::]:31337
[I][2019-11-29T00:26:09+0000] New connection from: [::ffff:94.25.172.81]:54548 on: [::ffff:192.168.144.2]:31337
[I][2019-11-29T00:26:09+0000] Executing '/var/task/main.py' for '[::ffff:94.25.172.81]:54548'
[I][2019-11-29T00:26:09+0000] pid=10 ([::ffff:94.25.172.81]:54548) exited with status: 2, (PIDs left: 0)
[I][2019-11-29T00:26:10+0000] New connection from: [::ffff:94.25.172.81]:28588 on: [::ffff:192.168.144.2]:31337
[I][2019-11-29T00:26:10+0000] Executing '/var/task/main.py' for '[::ffff:94.25.172.81]:28588'
[I][2019-11-29T00:26:10+0000] pid=11 ([::ffff:94.25.172.81]:28588) exited with status: 2, (PIDs left: 0)
[I][2019-11-29T00:27:28+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:27:28+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'bash', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:27:28+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:27:28+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:27:28+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:27:28+0000][63] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:27:28+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:27:28+0000][63] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[E][2019-11-29T00:27:28+0000][63] int net::getRecvSocket(const char*, int)():249 bind(host:[::] (orig:'::'), port:31337): Address already in use
[I][2019-11-29T00:27:30+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:27:30+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/bin/bash', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:27:30+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:27:30+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:27:30+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:27:30+0000][67] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:27:30+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:27:30+0000][67] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[E][2019-11-29T00:27:30+0000][67] int net::getRecvSocket(const char*, int)():249 bind(host:[::] (orig:'::'), port:31337): Address already in use
[I][2019-11-29T00:27:34+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:27:34+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/bin/bash', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:27:34+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:27:34+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:27:34+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:27:34+0000][71] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:27:34+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:27:34+0000][71] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[E][2019-11-29T00:27:34+0000][71] int net::getRecvSocket(const char*, int)():249 bind(host:[::] (orig:'::'), port:31337): Address already in use
[I][2019-11-29T00:30:47+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:30:47+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:30:47+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:30:47+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:30:47+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:30:47+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:30:47+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:30:47+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:30:47+0000] Listening on [::]:31337
[I][2019-11-29T00:31:31+0000] New connection from: [::ffff:192.168.160.1]:49228 on: [::ffff:192.168.160.2]:31337
[I][2019-11-29T00:31:31+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.160.1]:49228'
[I][2019-11-29T00:31:31+0000] pid=9 ([::ffff:192.168.160.1]:49228) exited with status: 2, (PIDs left: 0)
[I][2019-11-29T00:32:23+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:32:23+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:32:23+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:32:23+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:32:23+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:32:23+0000][9] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:32:23+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:32:23+0000][9] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:32:23+0000] Listening on [::]:31337
[I][2019-11-29T00:32:25+0000] New connection from: [::ffff:192.168.176.1]:39222 on: [::ffff:192.168.176.2]:31337
[I][2019-11-29T00:32:25+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.176.1]:39222'
[I][2019-11-29T00:32:29+0000] pid=10 ([::ffff:192.168.176.1]:39222) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T00:32:30+0000] New connection from: [::ffff:192.168.176.1]:39226 on: [::ffff:192.168.176.2]:31337
[I][2019-11-29T00:32:30+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.176.1]:39226'
[I][2019-11-29T00:32:36+0000] pid=11 ([::ffff:192.168.176.1]:39226) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T00:32:38+0000] New connection from: [::ffff:192.168.176.1]:39230 on: [::ffff:192.168.176.2]:31337
[I][2019-11-29T00:32:38+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.176.1]:39230'
[I][2019-11-29T00:32:40+0000] pid=12 ([::ffff:192.168.176.1]:39230) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T00:33:12+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:33:12+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:33:12+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:33:12+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:33:12+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:33:12+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:33:12+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:33:12+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:33:12+0000] Listening on [::]:31337
[I][2019-11-29T00:33:14+0000] New connection from: [::ffff:192.168.192.1]:47152 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T00:33:14+0000] Executing '/var/task/main.py' for '[::ffff:192.168.192.1]:47152'
[E][2019-11-29T00:33:14+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:33:14+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:33:14+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:33:14+0000] pid=9 ([::ffff:192.168.192.1]:47152) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:33:14+0000] New connection from: [::ffff:192.168.192.1]:47156 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T00:33:15+0000] Executing '/var/task/main.py' for '[::ffff:192.168.192.1]:47156'
[E][2019-11-29T00:33:15+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:33:15+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:33:15+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:33:15+0000] pid=10 ([::ffff:192.168.192.1]:47156) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:33:15+0000] New connection from: [::ffff:192.168.192.1]:47160 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T00:33:15+0000] Executing '/var/task/main.py' for '[::ffff:192.168.192.1]:47160'
[E][2019-11-29T00:33:15+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:33:15+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:33:15+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:33:15+0000] pid=11 ([::ffff:192.168.192.1]:47160) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:33:16+0000] New connection from: [::ffff:192.168.192.1]:47164 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T00:33:16+0000] Executing '/var/task/main.py' for '[::ffff:192.168.192.1]:47164'
[E][2019-11-29T00:33:16+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:33:16+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:33:16+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:33:16+0000] pid=12 ([::ffff:192.168.192.1]:47164) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:33:33+0000] New connection from: [::ffff:192.168.192.1]:47168 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T00:33:33+0000] Executing '/var/task/main.py' for '[::ffff:192.168.192.1]:47168'
[E][2019-11-29T00:33:33+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:33:33+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:33:33+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:33:33+0000] pid=13 ([::ffff:192.168.192.1]:47168) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:34:11+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:34:11+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:34:11+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:34:11+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:34:11+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:34:11+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:34:11+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:34:11+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:34:11+0000] Listening on [::]:31337
[I][2019-11-29T00:34:15+0000] New connection from: [::ffff:192.168.208.1]:39544 on: [::ffff:192.168.208.2]:31337
[I][2019-11-29T00:34:15+0000] Executing '/var/task/main.py' for '[::ffff:192.168.208.1]:39544'
[I][2019-11-29T00:34:17+0000] pid=9 ([::ffff:192.168.208.1]:39544) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T00:34:21+0000] New connection from: [::ffff:192.168.208.1]:39548 on: [::ffff:192.168.208.2]:31337
[I][2019-11-29T00:34:21+0000] Executing '/var/task/main.py' for '[::ffff:192.168.208.1]:39548'
[I][2019-11-29T00:34:43+0000] pid=10 ([::ffff:192.168.208.1]:39548) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T00:50:49+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:50:49+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:50:49+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:50:49+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:50:49+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:50:49+0000][9] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:50:49+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:50:49+0000][9] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:50:49+0000] Listening on [::]:31337
[I][2019-11-29T00:50:51+0000] New connection from: [::ffff:172.18.0.1]:34306 on: [::ffff:172.18.0.2]:31337
[W][2019-11-29T00:50:51+0000][1] bool mnt::mountPt(mount_t*, const char*, const char*)():203 mount(''/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true') src:'/chroot/' dstpath:'/tmp/nsjail.0.root//' failed: No such file or directory
[F][2019-11-29T00:50:51+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:50:51+0000][9] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:50:51+0000] pid=10 ([::ffff:172.18.0.1]:34306) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:51:27+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:51:27+0000] Jail parameters: hostname:'jail', chroot:'', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:51:27+0000] Mount: '/' flags:MS_RDONLY type:'tmpfs' options:'' dir:true
[I][2019-11-29T00:51:27+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:51:27+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:51:27+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:51:27+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:51:27+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:51:27+0000] Listening on [::]:31337
[I][2019-11-29T00:51:30+0000] New connection from: [::ffff:172.22.0.1]:48750 on: [::ffff:172.22.0.2]:31337
[I][2019-11-29T00:51:30+0000] Executing '/var/task/main.py' for '[::ffff:172.22.0.1]:48750'
[E][2019-11-29T00:51:30+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:51:30+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:51:30+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:51:30+0000] pid=9 ([::ffff:172.22.0.1]:48750) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:52:18+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:52:18+0000] Jail parameters: hostname:'jail', chroot:'', process:'/usr/bin/env', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:52:18+0000] Mount: '/' flags:MS_RDONLY type:'tmpfs' options:'' dir:true
[I][2019-11-29T00:52:18+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:52:18+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:52:18+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:52:18+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:52:18+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:52:18+0000] Listening on [::]:31337
[I][2019-11-29T00:52:20+0000] New connection from: [::ffff:172.24.0.1]:36220 on: [::ffff:172.24.0.2]:31337
[I][2019-11-29T00:52:20+0000] Executing '/usr/bin/env' for '[::ffff:172.24.0.1]:36220'
[E][2019-11-29T00:52:20+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/usr/bin/env') failed: No such file or directory
[F][2019-11-29T00:52:20+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:52:20+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:52:20+0000] pid=9 ([::ffff:172.24.0.1]:36220) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:52:21+0000] New connection from: [::ffff:172.24.0.1]:36224 on: [::ffff:172.24.0.2]:31337
[I][2019-11-29T00:52:21+0000] Executing '/usr/bin/env' for '[::ffff:172.24.0.1]:36224'
[E][2019-11-29T00:52:21+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/usr/bin/env') failed: No such file or directory
[F][2019-11-29T00:52:21+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:52:21+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:52:21+0000] pid=10 ([::ffff:172.24.0.1]:36224) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:52:58+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:52:58+0000] Jail parameters: hostname:'jail', chroot:'', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:52:58+0000] Mount: '/' flags:MS_RDONLY type:'tmpfs' options:'' dir:true
[I][2019-11-29T00:52:58+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:52:58+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:52:58+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:52:58+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:52:58+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:52:58+0000] Listening on [::]:31337
[I][2019-11-29T00:53:00+0000] New connection from: [::ffff:172.31.0.1]:54616 on: [::ffff:172.31.0.2]:31337
[I][2019-11-29T00:53:00+0000] Executing '/usr/bin/python2.7' for '[::ffff:172.31.0.1]:54616'
[E][2019-11-29T00:53:00+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/usr/bin/python2.7') failed: No such file or directory
[F][2019-11-29T00:53:00+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:53:00+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:53:00+0000] pid=9 ([::ffff:172.31.0.1]:54616) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:53:00+0000] New connection from: [::ffff:172.31.0.1]:54620 on: [::ffff:172.31.0.2]:31337
[I][2019-11-29T00:53:00+0000] Executing '/usr/bin/python2.7' for '[::ffff:172.31.0.1]:54620'
[E][2019-11-29T00:53:00+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/usr/bin/python2.7') failed: No such file or directory
[F][2019-11-29T00:53:00+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:53:00+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:53:00+0000] pid=10 ([::ffff:172.31.0.1]:54620) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:55:27+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:55:27+0000] Jail parameters: hostname:'jail', chroot:'/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:55:27+0000] Mount: '/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:55:27+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:55:27+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:55:27+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:55:27+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:55:27+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:55:27+0000] Listening on [::]:31337
[I][2019-11-29T00:55:32+0000] New connection from: [::ffff:192.168.64.1]:47212 on: [::ffff:192.168.64.2]:31337
[I][2019-11-29T00:55:32+0000] Executing '/var/task/main.py' for '[::ffff:192.168.64.1]:47212'
[I][2019-11-29T00:55:32+0000] pid=9 ([::ffff:192.168.64.1]:47212) exited with status: 127, (PIDs left: 0)
[I][2019-11-29T00:55:34+0000] New connection from: [::ffff:192.168.64.1]:47216 on: [::ffff:192.168.64.2]:31337
[I][2019-11-29T00:55:34+0000] Executing '/var/task/main.py' for '[::ffff:192.168.64.1]:47216'
[I][2019-11-29T00:55:34+0000] pid=10 ([::ffff:192.168.64.1]:47216) exited with status: 127, (PIDs left: 0)
[I][2019-11-29T00:56:02+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:56:02+0000] Jail parameters: hostname:'jail', chroot:'/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:56:02+0000] Mount: '/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:56:02+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:56:02+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:56:02+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:56:02+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:56:02+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:56:02+0000] Listening on [::]:31337
[I][2019-11-29T00:56:03+0000] New connection from: [::ffff:192.168.80.1]:41872 on: [::ffff:192.168.80.2]:31337
[I][2019-11-29T00:56:03+0000] Executing '/var/task/main.py' for '[::ffff:192.168.80.1]:41872'
[E][2019-11-29T00:56:03+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:56:03+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:56:03+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:56:03+0000] pid=9 ([::ffff:192.168.80.1]:41872) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:56:04+0000] New connection from: [::ffff:192.168.80.1]:41876 on: [::ffff:192.168.80.2]:31337
[I][2019-11-29T00:56:04+0000] Executing '/var/task/main.py' for '[::ffff:192.168.80.1]:41876'
[E][2019-11-29T00:56:04+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T00:56:04+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T00:56:04+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T00:56:05+0000] pid=10 ([::ffff:192.168.80.1]:41876) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T00:57:21+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:57:21+0000] Jail parameters: hostname:'jail', chroot:'/', process:'/bin/sh', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:57:21+0000] Mount: '/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:57:21+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:57:21+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:57:21+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:57:21+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:57:21+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:57:21+0000] Listening on [::]:31337
[I][2019-11-29T00:57:25+0000] New connection from: [::ffff:192.168.96.1]:39550 on: [::ffff:192.168.96.2]:31337
[I][2019-11-29T00:57:25+0000] Executing '/bin/sh' for '[::ffff:192.168.96.1]:39550'
[I][2019-11-29T00:57:27+0000] pid=9 ([::ffff:192.168.96.1]:39550) exited with status: 0, (PIDs left: 0)
[I][2019-11-29T00:57:46+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:57:46+0000] Jail parameters: hostname:'jail', chroot:'/', process:'/bin/sh', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:57:46+0000] Mount: '/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:57:46+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:57:46+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:57:46+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:57:46+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:57:46+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:57:46+0000] Listening on [::]:31337
[I][2019-11-29T00:57:48+0000] New connection from: [::ffff:192.168.112.1]:42092 on: [::ffff:192.168.112.2]:31337
[I][2019-11-29T00:57:48+0000] Executing '/bin/sh' for '[::ffff:192.168.112.1]:42092'
[I][2019-11-29T00:58:13+0000] pid=9 ([::ffff:192.168.112.1]:42092) exited with status: 0, (PIDs left: 0)
[I][2019-11-29T00:58:47+0000] Mode: LISTEN_TCP
[I][2019-11-29T00:58:47+0000] Jail parameters: hostname:'jail', chroot:'/', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T00:58:47+0000] Mount: '/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T00:58:47+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T00:58:47+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T00:58:47+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T00:58:47+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T00:58:47+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T00:58:47+0000] Listening on [::]:31337
[I][2019-11-29T00:58:49+0000] New connection from: [::ffff:192.168.144.1]:59370 on: [::ffff:192.168.144.2]:31337
[I][2019-11-29T00:58:49+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.144.1]:59370'
[I][2019-11-29T00:59:00+0000] pid=9 ([::ffff:192.168.144.1]:59370) exited with status: 0, (PIDs left: 0)
[I][2019-11-29T01:03:24+0000] Mode: LISTEN_TCP
[I][2019-11-29T01:03:24+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/var/task/main.py', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T01:03:24+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T01:03:24+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T01:03:24+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T01:03:24+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T01:03:24+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T01:03:24+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T01:03:24+0000] Listening on [::]:31337
[I][2019-11-29T01:03:29+0000] New connection from: [::ffff:192.168.176.1]:39452 on: [::ffff:192.168.176.2]:31337
[I][2019-11-29T01:03:29+0000] Executing '/var/task/main.py' for '[::ffff:192.168.176.1]:39452'
[E][2019-11-29T01:03:29+0000][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():200 execve('/var/task/main.py') failed: No such file or directory
[F][2019-11-29T01:03:29+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():449 Launching child process failed
[W][2019-11-29T01:03:29+0000][8] bool subproc::runChild(nsjconf_t*, int, int, int)():479 Received error message from the child process before it has been executed
[I][2019-11-29T01:03:29+0000] pid=9 ([::ffff:192.168.176.1]:39452) exited with status: 255, (PIDs left: 0)
[I][2019-11-29T01:05:01+0000] Mode: LISTEN_TCP
[I][2019-11-29T01:05:01+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T01:05:01+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T01:05:01+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T01:05:01+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T01:05:01+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T01:05:01+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T01:05:01+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T01:05:01+0000] Listening on [::]:31337
[I][2019-11-29T01:05:02+0000] New connection from: [::ffff:192.168.192.1]:47374 on: [::ffff:192.168.192.2]:31337
[I][2019-11-29T01:05:02+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.192.1]:47374'
[I][2019-11-29T01:05:02+0000] pid=9 ([::ffff:192.168.192.1]:47374) exited with status: 2, (PIDs left: 0)
[I][2019-11-29T01:05:25+0000] Mode: LISTEN_TCP
[I][2019-11-29T01:05:25+0000] Jail parameters: hostname:'jail', chroot:'/chroot/', process:'/usr/bin/python2.7', bind:[::]:31337, max_conns_per_ip:0, time_limit:30, personality:0, daemonize:false, clone_newnet:false, clone_newuser:true, clone_newns:true, clone_newpid:true, clone_newipc:true, clone_newuts:true, clone_newcgroup:true, keep_caps:false, disable_no_new_privs:false, max_cpus:0
[I][2019-11-29T01:05:25+0000] Mount: '/chroot/' -> '/' flags:MS_RDONLY|MS_BIND|MS_REC|MS_PRIVATE type:'' options:'' dir:true
[I][2019-11-29T01:05:25+0000] Mount: '/proc' flags:MS_RDONLY type:'proc' options:'' dir:true
[I][2019-11-29T01:05:25+0000] Uid map: inside_uid:65534 outside_uid:0 count:1 newuidmap:false
[W][2019-11-29T01:05:25+0000][8] void cmdline::logParams(nsjconf_t*)():254 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[I][2019-11-29T01:05:25+0000] Gid map: inside_gid:65534 outside_gid:0 count:1 newgidmap:false
[W][2019-11-29T01:05:25+0000][8] void cmdline::logParams(nsjconf_t*)():264 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files
[I][2019-11-29T01:05:25+0000] Listening on [::]:31337
[I][2019-11-29T01:05:26+0000] New connection from: [::ffff:192.168.208.1]:39746 on: [::ffff:192.168.208.2]:31337
[I][2019-11-29T01:05:26+0000] Executing '/usr/bin/python2.7' for '[::ffff:192.168.208.1]:39746'
[I][2019-11-29T01:05:29+0000] pid=9 ([::ffff:192.168.208.1]:39746) exited with status: 1, (PIDs left: 0)
[I][2019-11-29T01:07:55+0000] New connection from: [::ffff:94.25.172.81]:18180 on: [::ffff:192.168.208.2]:31337
[I][2019-11-29T01:07:55+0000] Executing '/usr/bin/python2.7' for '[::ffff:94.25.172.81]:18180'
[I][2019-11-29T01:07:57+0000] New connection from: [::ffff:94.25.172.81]:28249 on: [::ffff:192.168.208.2]:31337
[I][2019-11-29T01:07:57+0000] Executing '/usr/bin/python2.7' for '[::ffff:94.25.172.81]:28249'
[I][2019-11-29T01:07:57+0000] pid=10 ([::ffff:94.25.172.81]:18180) exited with status: 0, (PIDs left: 1)
[I][2019-11-29T01:08:02+0000] pid=11 ([::ffff:94.25.172.81]:28249) exited with status: 1, (PIDs left: 0)
